tests/upgrade_script/static/prometheus_service_monitors.input.yaml

```diff
@@ -1,59 +1,59 @@
 prometheus-operator:
```

Comment:
I don't see a point of changing this file

Reply:
Done, reverted